packages/hathor-rpc-handler/src/rpcMethods/getConnectedNetwork.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Network } from '@hathor/wallet-lib';
+import { HathorWallet } from '@hathor/wallet-lib';
+import { GetConnectedNetworkRpcRequest } from '../types';
+
+/**
+ * Handles the 'get_connected_network' RPC request by retrieving the network information
+ * from the wallet and returning the network name and genesis hash.
+ * 
+ * @param rpcRequest - The RPC request object containing the method and parameters.
```

Comment:
suggestion: Inform that this parameter is unused, also the parameter name in the jsdocs doesn't match the one on the actual function.

Aren´t any of the static code analyzers complaining about this? Maybe the linting rules could be adjusted for this, so that future jsdocs are as close as possible to the code.

Reply:
I would have to add a plugin to eslint, would rather not for now

Refactored the docs to inform the parameter is unused and the param name, thanks!